packages/react-native-tracer-provider/src/EmbraceNativeSpan.ts

```diff
@@ -0,0 +1,224 @@
+import {
+  Attributes,
+  AttributeValue,
+  Exception,
+  Span,
+  SpanContext,
+  SpanStatus,
+  SpanStatusCode,
+  TimeInput,
+} from '@opentelemetry/api';
+import { Link } from '@opentelemetry/api/build/src/trace/link';
+import { TracerProviderModule } from './TracerProviderModule';
+import { SpanContextSyncBehaviour } from './types';
+import {
+  isAttributes,
+  normalizeAttributes,
+  normalizeLinks,
+  normalizeTime,
+} from './util';
+
+/**
+ * EmbraceNativeSpan implements a Span over the native Embrace Android and iOS SDKs.
+ *
+ * A handful of simple attributes are maintained on the JS side for each span include a unique ID based on the
```

Comment:
I feel like this should be `including` instead of `include`

Reply:
yep